website/source/docs/guides/creating-certificates.html.md

```diff
@@ -192,7 +193,6 @@ $ export CONSUL_CLIENT_KEY=cli-key.pem
 After these environment variables are correctly configured, the CLI will
 respond as expected.
 
-[cfssl]: https://cfssl.org/
 [letsencrypt]: https://letsencrypt.org/
 [vault]: https://www.vaultproject.io/
 [vault-pki]: https://www.vaultproject.io/docs/secrets/pki/index.html
```

Comment:
If you have time and a good sense of next steps, it would be good to add a final section. We haven't been super consistent with "Next Steps" sections in out guides but I do think they are beneficial for learners. If you don't have next steps, even a summary would be nice!

Reply:
I wrote a tiny summary with next steps!